lib/welcome-view.coffee

```diff
@@ -55,6 +55,9 @@ class WelcomeView extends ScrollView
       eventName = $(this).attr('data-event')
       Reporter.sendEvent("clicked-welcome-#{eventName}-link") if eventName
 
+    @on 'click', 'a.js-metrics', ->
+      atom.workspace.open('atom://config/packages/metrics')
```

Comment:
Nothing, rather than looking into it I just added a click event. I didn't want to dwell on it too much as I figured if we address "open from link in browser" we'd have to handle it then.